package.json

```diff
@@ -17,6 +17,7 @@
     "@wordpress/i18n": "^3.14.0",
     "@wordpress/scripts": "^12.1.1",
     "copy-webpack-plugin": "^6.0.1",
+    "govuk-frontend": "^3.7.0",
     "imagemin-mozjpeg": "^9.0.0",
```

Comment:
If its in dependences do we need it in devdependences?

Reply:
Ah - we do not! I'll remove it :)

Reply:
Now removed :)